romdisk/uart.asm

```diff
@@ -115,7 +117,9 @@ uartsnd_main:
         dec bc  ; Do not count the command itself
         ld a, b
         or c
+        jr z, _uartsnd_usage
         ret z
```

Comment:
This instruction can be removed since `jr z` is just before

Reply:
Which command did you mean? `jr z, uartsnd_usage` or `ret z` ?

Reply:
#fixed